barretenberg/cpp/src/barretenberg/honk/instance/instances.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+#include "barretenberg/honk/instance/prover_instance.hpp"
+#include "barretenberg/honk/instance/verifier_instance.hpp"
+namespace proof_system::honk {
+
+template <typename Flavor_, size_t NUM_> struct ProverInstances {
```

Comment:
this is a duplication of your code in the combiner reworked to fit the rest of the interfaces